### DISCUSSION TITLE
can not load videos

### DISCUSSION THREAD

**Discussion by Kyle-1006** [i-TechX/iTechX#100] [Q&A]

**Comment by why-in-Shanghaitech** [comment]
Hi! Thank you for reaching out. Unfortunately, we've heard that the our video hosting and streaming platform, streamsb, is offline (#97). I'm not sure whether it is under temporary maintenance or shutdown forever. I'll keep an eye on this and see if we need to remove all the video resources.

According to the record, most videos are never played. So if the video platform won't recover, I need to make a decision whether to upload them somewhere else again. This might take a lot of effort. What do you think? Don't hesitate to tell me if you have any ideas.

**Comment by why-in-Shanghaitech** [comment]
Sadly, it has been confirmed that streamsb has been closed on July 30.

https://www.wjunction.com/threads/streamsb-closes-for-good-july-30.261500/

iTechX needs to remove all video resources. See #97 for details.

Duplicate for #101. I'm closing this discussion for now.
